### DISCUSSION TITLE
Can it stop bugging me about making backups?

### DISCUSSION THREAD

**Discussion by engerim42** [Authenticator-Extension/Authenticator#1054] [Ideas]
Could be an option. I do take care of my own backup of the hashes.

**Comment by Sneezry** [comment]
Can you share more information? What specific problems have you encountered?

**Reply by engerim42** [reply]
The "remind_backup" function is nagging me once in a while to make backups (a popup appears).
But there is no option to disable this warning popup.
Its especially annoying as I always got to click it away at the moment I need a code.
I made a file backup already a while ago and didn't add any new codes.

```
# grep -n remind_backup ./src/popup.ts
217:          instance.i18n.remind_backup
254:          instance.i18n.remind_backup
291:          instance.i18n.remind_backup
302:    instance.$store.commit("notification/alert", instance.i18n.remind_backup);
```

**Reply by Sneezry** [reply]
Our notification is programmed to appear every 30 days. Have you noticed that it's showing up as expected on your end?

https://github.com/Authenticator-Extension/Authenticator/blob/c5976b63c5ff3b8c6a1e2ba5077071abf973aead/src/popup.ts#L111-L112

If it's not appearing as scheduled, it's possible that your browsing history may have been cleared. Our extension saves the date of the last notification in your local storage, so if that gets cleared, the extension will prompt you again.

**Reply by engerim42** [reply]
Yes, definitively more often (which is likely not as expected). How could I submit a feature request that backup reminders are completely disabled?

**Reply by Sneezry** [reply]
While it may seem like a good idea to completely disable backup reminders, I believe it's important to exercise caution. If the reminders are disabled by mistake, users may forget to backup their secrets. Compared to other platforms, web browsers are particularly susceptible to data loss due to various reasons such as browser crashes, data being lost after a browser update, accidentally uninstalling the extension, or clearing browser data, among other things.

**Reply by engerim42** [reply]
true, but it always nags the user when they clicked the icon to get a code. Maybe the way the reminders are done could be done differently, such as displaying a small box between the "Search" bar and the Auth codes, not a popup.
